### PR TITLE
feat: append a 2-digit CRC to board ids to prevent dangerous typos

### PR DESCRIPTION
**Pull request by michaelfig**
This helps ensure you get the board ID that you meant.

### CODE REVIEW

File: packages/cosmic-swingset/lib/ag-solo/vats/lib-board.js [hunk 1]
```diff
@@ -16,7 +38,8 @@ function makeBoard(seed = 0) {
         // Retry until we have a unique id.
         let id;
         do {
-          id = sparseInts.next().value.toString();
+          const num = sparseInts.next().value;
+          id = `${num.toString()}${calcCrc(num)}`;
```
Comment (FUDCo) [inline packages/cosmic-swingset/lib/ag-solo/vats/lib-board.js]: Curious why `${num.toString()}` instead of `${num}`?

**Review by FUDCo — APPROVED** [review]
Holy credit card number, Batman!

File: packages/cosmic-swingset/lib/ag-solo/vats/lib-board.js (hunk 1 above)
Comment (michaelfig) [reply packages/cosmic-swingset/lib/ag-solo/vats/lib-board.js]: Cargo culting. :smile:

It's fixed now.

**Comment by katelynsills** [comment]
Nice!

**Pull request merged**